hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -384,6 +398,7 @@ private synchronized void flushWrittenBytesToServiceInternal(final long offset,
    * operation FIFO queue.
    */
   private synchronized void shrinkWriteOperationQueue() throws IOException {
+    outputStreamStatistics.queueShrinked();
```

Comment:
I don't understand the benifits of capturing this metric?

Reply:
it would also be "queueShrunk" as a name

Reply:
It is an expensive method. So, I thought it would benefit in knowing how many times it's being called after some write operations. 
Sorry for the spelling mistake.